### ISSUE TITLE
Vagrant 2.0.2 breaks loading of libffi.so.5 (worked with 2.0.1)

### ISSUE THREAD

**Issue by zimmski** [hashicorp/vagrant#9445]
### Vagrantfile
```ruby
# You can use any Vagrantfile with the following addition

config.trigger.after :up do
	run "vagrant fsnotify"
end
```

### Debug output
```
==> default: /opt/vagrant/embedded/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require': libffi.so.5: cannot open shared object file: No such file or directory - /opt/vagrant/embedded/gems/gems/ffi-1.9.18/lib/ffi_c.so (LoadError)
==> default:    from /opt/vagrant/embedded/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
==> default:    from /opt/vagrant/embedded/gems/gems/ffi-1.9.18/lib/ffi.rb:6:in `rescue in <top (required)>'
==> default:    from /opt/vagrant/embedded/gems/gems/ffi-1.9.18/lib/ffi.rb:3:in `<top (required)>'
==> default:    from /opt/vagrant/embedded/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
==> default:    from /opt/vagrant/embedded/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
==> default:    from /opt/vagrant/embedded/gems/gems/rb-inotify-0.9.10/lib/rb-inotify/native.rb:1:in `<top (required)>'
==> default:    from /opt/vagrant/embedded/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
==> default:    from /opt/vagrant/embedded/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
==> default:    from /opt/vagrant/embedded/gems/gems/rb-inotify-0.9.10/lib/rb-inotify.rb:2:in `<top (required)>'
==> default:    from /opt/vagrant/embedded/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
==> default:    from /opt/vagrant/embedded/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
==> default:    from /opt/vagrant/embedded/gems/gems/listen-3.1.5/lib/listen/adapter/linux.rb:30:in `_configure'
==> default:    from /opt/vagrant/embedded/gems/gems/listen-3.1.5/lib/listen/adapter/base.rb:45:in `block in configure'
==> default:    from /opt/vagrant/embedded/gems/gems/listen-3.1.5/lib/listen/adapter/base.rb:40:in `each'
==> default:    from /opt/vagrant/embedded/gems/gems/listen-3.1.5/lib/listen/adapter/base.rb:40:in `configure'
==> default:    from /opt/vagrant/embedded/gems/gems/listen-3.1.5/lib/listen/adapter/base.rb:63:in `start'
==> default:    from /opt/vagrant/embedded/gems/gems/listen-3.1.5/lib/listen/listener.rb:68:in `block in <class:Listener>'
==> default:    from /opt/vagrant/embedded/gems/gems/listen-3.1.5/lib/listen/fsm.rb:121:in `instance_eval'
==> default:    from /opt/vagrant/embedded/gems/gems/listen-3.1.5/lib/listen/fsm.rb:121:in `call'
==> default:    from /opt/vagrant/embedded/gems/gems/listen-3.1.5/lib/listen/fsm.rb:91:in `transition_with_callbacks!'
==> default:    from /opt/vagrant/embedded/gems/gems/listen-3.1.5/lib/listen/fsm.rb:57:in `transition'
==> default:    from /opt/vagrant/embedded/gems/gems/listen-3.1.5/lib/listen/listener.rb:91:in `start'
==> default:    from /home/zimmski/.vagrant.d/gems/2.4.3/gems/vagrant-fsnotify-0.3.0/lib/vagrant-fsnotify/command-fsnotify.rb:127:in `block in execute'
==> default:    from /opt/vagrant/embedded/gems/gems/vagrant-2.0.2/lib/vagrant/util/busy.rb:19:in `busy'
==> default:    from /home/zimmski/.vagrant.d/gems/2.4.3/gems/vagrant-fsnotify-0.3.0/lib/vagrant-fsnotify/command-fsnotify.rb:126:in `execute'
==> default:    from /opt/vagrant/embedded/gems/gems/vagrant-2.0.2/lib/vagrant/cli.rb:46:in `execute'
==> default:    from /opt/vagrant/embedded/gems/gems/vagrant-2.0.2/lib/vagrant/environment.rb:269:in `cli'
==> default:    from /opt/vagrant/embedded/gems/gems/vagrant-2.0.2/bin/vagrant:151:in `<main>'
```

### Expected behavior
The ffi_c.so should be loaded and the plugin should be executed as usual.

### Actual behavior
The load error (see above) appears and the Vagrant instance is not usable. PLEASE NOTE: This works with 2.0.1. So this problem regresses with a bugfix version.

### Steps to reproduce
1. Use the fsnotify plugin
2. vagrant up
3. See the load error

**Comment by chrisroberts** [comment]
Hi there,

This should be resolved in the 2.0.3 release. Can you confirm if you are still encountering this error? Thanks!

**Comment by zimmski** [comment]
Works with 2.0.3, thanks! Which commit fixed that issue?

**Comment by chrisroberts** [comment]
@zimmski It was an issue with the installers build and fixed in hashicorp/vagrant-installers#130
